index.html

```diff
@@ -184,6 +184,27 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
       </div>
     </section>
   </div>
+  <section class="contact-section">
+    <div class="contact-content">
+      <h2 class="title contact-text contact-title">Contact me</h2>
+      <p class="contact-text">If you have an application you are interested in developing, a feature that you need built
+        or a project that needs coding. I’d love to help with it </p>
+      <form id="fs-frm" name="simple-contact-form" accept-charset="utf-8" action="https://formspree.io/f/mnqlrawl"
+        method="post" class="contact-form">
+        <fieldset id="fs-frm-inputs">
+          <label for="full-name">Full Name</label>
+          <input type="text" maxlength="30" name="name" id="full-name" placeholder="First and Last Name" required="">
+          <label for="email-address">Email Address</label>
+          <input type="email" name="_replyto" id="email-address" placeholder="Email Address" required="">
+          <label for="message">Message</label>
+          <textarea rows="5" name="message" id="message" placeholder="Write your message here." required=""></textarea>
```

Comment:
Please note that required is an attribute without any value, so it should be only `required`.